websocket/backend/app/controllers/api/v1/todos_controller.rb

```diff
@@ -0,0 +1,61 @@
+class Api::V1::TodosController < ApplicationController
+  before_action :set_todo, only: %i[ show edit update destroy ]
+  skip_before_action :verify_authenticity_token
+
+  # GET /todos or /todos.json
+  def index
+    @todos = Todo.all
+
+    render json: @todos
+  end
+
+  # GET /todos/1 or /todos/1.json
+  def show
+    render json: @todos
+  end
+
+  # GET /todos/new
+  def new
+    @todo = Todo.new
+  end
+
+  # GET /todos/1/edit
+  def edit
+  end
+
+  # POST /todos or /todos.json
+  def create
+    @todo = Todo.new(todo_params)
+
+    if @todo.save
+      render json: @todo, status: :created, location: api_v1_todos_path(@todo)
+    else
+      render json: @todo.errors, status: :unprocessable_entity
+    end
+  end
+
+  # PATCH/PUT /todos/1 or /todos/1.json
+  def update
+    if @todo.update(todo_params)
+      render json: @todo
+    else
+      render json: @todo.errors, status: :unprocessable_entity
+    end
+  end
+
+  # DELETE /todos/1 or /todos/1.json
+  def destroy
+    @todo.destroy
+  end
+
+  private
+  # Use callbacks to share common setup or constraints between actions.
+  def set_todo
+    @todo = Todo.find(params[:id])
+  end
+
+  # Only allow a trusted parameter "white list" through.
+  def todo_params
+    params.require(:todo).permit(:task, :body)
```

Comment:
This don't match your migration file at all.

Reply:
forgot to run rails:db migrate after paste back 😅